### DISCUSSION TITLE
Open port for external access

### DISCUSSION THREAD

**Discussion by schtoffer** [gurgleapps/pico-web-server-control#8] [General]
Hi, I have a webserver on an ubuntu machine which I successfully have been able to set up port forwarding on my router. The page loads when I call just my external IP address. 

I am attempting to do just the same set up for the Pico running your software pico-web-server-control. I.e. Prior to this step, I can confirm that the Pico works just fine, running your software, when I call the IP-address for my local network.

I delete my other 80 port-forwarding which was associated to the said ubuntu server, and create a new port forwarding for the Raspberry pico. However, when I visit my external ip, nothing loads. I've tried to call [my external ip]/set-delay/0.1 from my iPhone on 3G. Is this the right way to go about it? Or is it something else I need to configure?

**Comment by mirkin** [comment]
Yes sounds correct, get the ip of the pico and do the same you did for your ubuntu. 
It should work just the same as it did for your ubuntu machine. 

I can't think of a reason it wouldn't work, I guess check it's not still sending to the ubuntu, try running the ubuntu and pico simultaneously on different ports.

Also, there is an alternative to port forwarding [here](https://gurgleapps.com/learn/coding/access-home-web-server-anywhere-guide) it may work for you.

**Comment by schtoffer** [comment]
Hi, thx!

Both methods worked for me after a lot of head-scratching. Turns out as for the first method, my router had a bug. Even when I set up port 80 for internal ip address of the pico, it actually set it up for another pico on my local network. Must be some cashed data that causes this faulty behaviou. Anyway, I got it to work eventually.
